lib/active_merchant/billing/gateways/mundipagg.rb

```diff
@@ -174,7 +174,9 @@ def add_credit_card(post, payment, options)
           post[:payment][:credit_card][:card][:exp_month] = payment.month
           post[:payment][:credit_card][:card][:exp_year] = payment.year
           post[:payment][:credit_card][:card][:cvv] = payment.verification_value
-          post[:payment][:credit_card][:card][:billing_address] = add_billing_address(options)
+          if options[:billing_address] || options[:address]
```

Comment:
A bit nitpicky, but it would be a little more compact to just call the `add_billing_address` method here, and put the conditional and [:billing_address] field creation into that method.